### ISSUE TITLE
Exclude map files from extension bundle

### ISSUE THREAD

**Issue by mjbvz** [microsoft/pylance-release#3647]
ms-python.vscode-pylance-2022.11.30

Hi, I noticed that a few map files snuck into the extension bundle:

```
1.0M	./dist/vendor.bundle.js.map
1.5M	./dist/pyright.bundle.js.map
2.5M	total
```

These should likely be removed, especially since the extension does not include the original source

**Comment by judej** [comment]
Triage Q: Will removing this break telemetry logging?

**Comment by mjbvz** [comment]
Since the original TS files are not included in the extension afaik, this should likely not change call stacks

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.1.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023131-18-January-2023-PreRelease)
